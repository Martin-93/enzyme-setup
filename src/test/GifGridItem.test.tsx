import "@testing-library/jest-dom";
import React from "react";
import GifGriditem from "../components/GifGridItem";
import { shallow } from "../setupTest";
//import { shallow, configure } from "enzyme";

describe("Pruebas GifGridItem", () => {
  test("should be retrun the correct component", () => {
    const wrapper = shallow(<GifGriditem title="" url="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
