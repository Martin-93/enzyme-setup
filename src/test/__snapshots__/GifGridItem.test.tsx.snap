// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas GifGridItem should be retrun the correct component 1`] = `ShallowWrapper {}`;
